corehq/apps/users/permissions.py

```diff
@@ -94,3 +94,11 @@ def _can_manage_releases_for_all_apps(couch_user, domain):
         domain, get_permission_name(Permissions.manage_releases),
         restrict_global_admin=True
     )
+
+
+def can_manage_releases_by_build_profile(couch_user, domain):
+    from corehq.apps.users.decorators import get_permission_name
+    from corehq.apps.users.models import Permissions
+    is_toggle_enabled = toggles.RELEASE_BUILDS_PER_PROFILE.enabled(domain)
+    can_edit_app = couch_user.has_permission(domain, get_permission_name(Permissions.edit_apps))
```

Comment:
I dont think we need this check.
`can_manage_releases` already means the user has access to this app.
we just just need to check for toggle
`{% if can_manage_releases and request|toggle_enabled:"RELEASE_BUILDS_PER_PROFILE" %}` should do

Reply:
Ohh!! Cool.

Reply:
Hey @mkangia 
Is it possible that user does not have edit access to the app but can manage releases?

Reply:
we should replicate whatever permission is on the view for the link.
(offlined that edit app should be checked here)

Reply:
I have actually done it with a different function. Is there any way to do it on the template itself or it is fine?

Reply:
You can probably just have `can_edit_app` in the context and check for toggle in the template. That will help get rid of adding a new method. Also, worth checking if `can_edit_app` is already called somewhere before this and reuse that.